virttest/utils_net.py

```diff
@@ -3705,6 +3706,32 @@ def _process_output(cmd, reg_pattern):
                                "mac %s" % mac_address)
 
 
+def get_linux_iface_info(mac, session=None):
+    """
+    Get host/guest info of certain interface with given mac address
```

Comment:
I final nitpick and I am happy: can you add a full stop here?
```diff
-Get host/guest info of certain interface with given mac address
+Get host/guest info of certain interface with given mac address.
```

Reply:
Of course :grin: